.ci_support/linux_64_openssl3.yaml

```diff
@@ -15,33 +15,17 @@ cxx_compiler:
 cxx_compiler_version:
 - '9'
 docker_image:
-- quay.io/condaforge/linux-anvil-comp7
+- quay.io/condaforge/linux-anvil-cos7-x86_64
 libffi:
 - '3.4'
 ncurses:
 - '6.2'
 openssl:
 - '3'
 pin_run_as_build:
-  bzip2:
-    max_pin: x
-  libffi:
-    max_pin: x.x
-  ncurses:
-    max_pin: x.x
   python:
     min_pin: x.x
     max_pin: x.x
-  readline:
-    max_pin: x
-  sqlite:
-    max_pin: x
-  tk:
-    max_pin: x.x
-  xz:
-    max_pin: x.x
-  zlib:
-    max_pin: x.x
```

Comment:
Wouldn't we also need to match the pins for previous `pin_run_as_build` packages to "period-appropriate" versions in the local CBC?